boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2020-2022 Nordic Semiconductor ASA
```

Comment:
How come only a subset of the files have Copyright from 20XX-2022, and some are just 2019?

Reply:
This is the same format as nrf5340dk_nrf5340. Though, I believe this is the old way, and now just the "first year" should be used.  For the CI building, can you comment @anangl or @nordicjm ?

Reply:
Possibly we can just add `nrf5340_audio_dk_nrf5340_cpuapp` to some of of the `sample.yaml` files we have in the `platform_allow` or `integration_platforms` like what we've done with `nrf5340dk_nrf5340_cpuapp`

Reply:
Done